bundles/org.openhab.binding.boschshc/README.md

```diff
@@ -228,6 +228,13 @@ This certificate is used for pairing between the Bridge and the Bosch Smart Home
 
 _Press and hold the Bosch Smart Home Controller Bridge button until the LED starts blinking after you save your settings for pairing_.
 
+### Supported Channels
+
+| Channel Type ID    | Item Type            | Writable | Description                                                        |
```

Comment:
It's technically correct, but more relevant for a user to know channel ids:
```suggestion
| Channel ID         | Item Type            | Writable | Description                                                        |
```

Reply:
I'm fine with using the term _ Channel ID_  instead of _Channel Type ID_, but then we should use it consistently throughout the whole file. I propose to change the other table headings in another PR, because this PR is already quite large and there are other things that should be made consistent in the readme.

Reply:
I changed it back to _Channel Type ID_ to be consistent in the file and it can be changed in a later PR in the whole file.